CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## UNRELEASED
+## 0.49.1 (November 14, 2022)
```

Comment:
Looks good, do we need to bump the version here:

https://github.com/hashicorp/consul-k8s/blob/9ed2eee66ebae96f1ce90ef7bd8c22c018a296a7/.github/workflows/test.yml#L7-L8

Reply:
good call

Reply:
after bumping this, ran into some test failures which i've fixed here https://github.com/hashicorp/consul-k8s/pull/1720